docs/content/guides/installation/install/index.md

```diff
@@ -0,0 +1,255 @@
+---
+type: docs
+title: "How-To: Install Radius on Kubernetes"
+linkTitle: "Install"
+description: "Learn how to install Radius on Kubernetes"
+weight: 100
+categories: "How-To"
+tags: ["Kubernetes"]
+aliases: 
+- /guides/operations/kubernetes/kubernetes-install
+---
+
+This guide goes through all the installation options and client tools to interact with Radius.
+
+## Radius CLI 
+
+The `rad` CLI is the primary interface for installing and operating Radius. Install it on any workstation or automation runner that interacts with Radius.
+
+Use the project installer to add `rad` plus the embedded `rad-bicep` compiler:
+
+```bash
+curl -fsSL "https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.sh" | /bin/bash
+```
+
+To install a **specific version**, pass the version number (without the leading `v`) to the script. The installer translates it to the tagged release:
+
+  ```bash
+  curl -fsSL "https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.sh" | /bin/bash -s 0.50.0
+  ```
+
+To install into **an alternate directory**, set `RADIUS_INSTALL_DIR` before invoking the script. This is required in environments like Azure Cloud Shell that disallow writes to `/usr/local/bin`:
+
+```bash
+export RADIUS_INSTALL_DIR=$HOME/bin
+curl -fsSL "https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.sh" | /bin/bash
+```
+
+On Windows, run the PowerShell installer shipped with the repository:
+
+```powershell
+irm https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.ps1 | iex
+```
```

Comment:
This can be confirmed by @brooke-hamilton or @DariuszPorowski (he may not have access to this repo)